concepts/usersession.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: User Session
+---
+
+A user session can be created to track interactions with you LLM app.
+Those interactions are visible in the analytics tab of the platform.
+
+<Frame caption="Chainlit analytics page">
+  <img src="/images/analytics.png" />
+</Frame>
+
+You can simply create a session with:
+
+<CodeGroup>
+
+```python
+from chainlit_client import ChainlitClient
+
+client = ChainlitClient()
+
+await client.api.create_user_session(is_interactive=True, anon_participant_identifier="Alex")
```

Comment:
I would give two examples
1. with an authenticated user
2. with anon_participant identifier
and explain the difference